### PR TITLE
Update sbt docs to better explain how to manually install Metals prior to 0.7.6 version

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalameta/metals/issues/2023

### CODE REVIEW

File: docs/editors/overview.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ specific sections.
 
 ```
 
+
```
Comment (tgodzik) [inline docs/editors/overview.md:16]: This is not rendering correctly now.

**Comment by kpbochenek** [comment]
Can't we simply drop 0.7.6 support and all that metalsEnable stuff? normally it is not even available with automatic installation

**Comment by tgodzik** [comment]
> Can't we simply drop 0.7.6 support and all that metalsEnable stuff? normally it is not even available with automatic installation

It's mostly for historic reasons, I am OK with dropping it now. Any other opinions?

**Comment by ckipp01** [comment]
> It's mostly for historic reasons, I am OK with dropping it now. Any other opinions?

I'm sort of for dropping it. I sort of think it may just cause confusion.

**Comment by tgodzik** [comment]
@ckipp01 @kpbochenek I removed the section altogether

**Review by kpbochenek — APPROVED** [review]
LGTM :+1:

**Pull request merged**